src/config.ts

```diff
@@ -415,6 +415,32 @@ export const prebuiltAppConfig: AppConfig = {
         context_window_size: 4096,
       },
     },
+    {
+      model: "https://huggingface.co/mlc-ai/Hermes-3-Llama-3.1-8B-q4f32_1-MLC",
+      model_id: "Hermes-3-Llama-3.1-8B-q4f32_1-MLC",
+      model_lib:
+        modelLibURLPrefix +
+        modelVersion +
+        "/Llama-3-8B-Instruct-q4f32_1-ctx4k_cs1k-webgpu.wasm",
```

Comment:
Let us use `llama-3.1` instead of `llama-3`. These two should be different due to their `rope_scaling` if we look at the `config.json`:
- Llama-3: https://huggingface.co/meta-llama/Meta-Llama-3-8B/blob/main/config.json
- Llama-3.1: https://huggingface.co/meta-llama/Meta-Llama-3.1-8B-Instruct/blob/main/config.json
- Hermes-3:  https://huggingface.co/NousResearch/Hermes-3-Llama-3.1-8B/blob/main/config.json

Reply:
fixed!